Microsoft.Toolkit.Wpf.UI.Controls/MapControl/MapControl.cs

```diff
@@ -77,7 +77,6 @@ protected override void OnInitialized(EventArgs e)
             Bind(nameof(ZoomLevel), ZoomLevelProperty, windows.UI.Xaml.Controls.Maps.MapControl.ZoomLevelProperty);
             Bind(nameof(Center), CenterProperty, windows.UI.Xaml.Controls.Maps.MapControl.CenterProperty, new WindowsXamlHostWrapperConverter());
             Bind(nameof(LoadingStatus), LoadingStatusProperty, windows.UI.Xaml.Controls.Maps.MapControl.LoadingStatusProperty, new WindowsXamlHostWrapperConverter());
-            Bind(nameof(MapElements), MapElementsProperty, windows.UI.Xaml.Controls.Maps.MapControl.MapElementsProperty);
```

Comment:
In the e-mail thread, the Layers property was noted as having the same bug.  Please fix for both.